+siibra/+core/Parcellation.m

```diff
@@ -1,32 +1,32 @@
 classdef Parcellation < handle
     properties
-        Id
-        Name
-        Atlas
-        Modality
-        Description
-        Graph
-        Spaces
+        id
+        name
+        atlas
+        modality
+        desciption
+        regionTree
+        spaces
     end
 
     methods
         function parcellation = Parcellation(parcellation_json, atlas)
-            parcellation.Id = strcat(parcellation_json.id.kg.kgSchema, '/', parcellation_json.id.kg.kgId);
-            parcellation.Name = parcellation_json.name;
-            parcellation.Atlas = atlas;
-            parcellation.Modality = parcellation_json.modality;
+            parcellation.id = strcat(parcellation_json.id.kg.kgSchema, '/', parcellation_json.id.kg.kgId);
+            parcellation.name = parcellation_json.name;
+            parcellation.atlas = atlas;
```

Comment:
Do you expect identifiers (strings?) or object references for such relationships?

Reply:
Likewise, prefer to start with typical MATLAB style (capitalized property names) and see how users react

Where there's a corresponding object type defined, e.g. Atlas, I'd suggest these to be object references.

Reply:
Yes, I am in favor of object references as well.